Childrens-Social-Care-CPD/Views/Shared/_Layout.cshtml

```diff
@@ -7,12 +7,11 @@
      <title>@ViewData["Title"]</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
     <!--[if !IE 8]><!-->
-    <link rel="stylesheet" href="/stylesheets/govuk-frontend-4.4.1.min.css">
+    <link rel="stylesheet" href="/css/application.css">
```

Comment:
Why we have added this file? this was added from prototype and we are not using it now

Reply:
ok, so its compiled sass file for  Govuk-frontend  and custom css ?